src/content/configuration/optimization.md

```diff
@@ -48,6 +48,8 @@ T> Learn how [mode](/configuration/mode/) works.
 
 Allows you to override the default minimizer by providing a different one or more customized [TerserPlugin](/plugins/terser-webpack-plugin/) instances.
 
+`"..."` can be used in `optimization.minimizer` to access the defaults.
```

Comment:
This is asking for an example when i read it. What do you think if we move it to a place below existing examples and add one showcasing accessing the defaults via ...?

Reply:
Sounds good 👍